mscore/shortcut.cpp

```diff
@@ -214,17 +214,17 @@ Shortcut Shortcut::_sc[] = {
          Icons::paste_ICON,
          Qt::ApplicationShortcut
          },
-    {
-       MsWidget::SCORE_TAB,
-       STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT | STATE_TEXT_EDIT |STATE_LYRICS_EDIT
-          | STATE_HARMONY_FIGBASS_EDIT,
-       "swap",
-       QT_TRANSLATE_NOOP("action","Swap Clipboard and Selection"),
-       0,
-       0,
-       Icons::swap_ICON,
-       Qt::ApplicationShortcut
-       },
+      {
+         MsWidget::SCORE_TAB,
+         STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT | STATE_TEXT_EDIT |STATE_LYRICS_EDIT
+            | STATE_HARMONY_FIGBASS_EDIT,
+         "swap",
+         QT_TRANSLATE_NOOP("action","Swap with Clipboard"),
```

Comment:
Or is this where the merge conflict comes from, as that part had been merged earlier already?